src/__tests__/utils/filterUtils.test.ts

```diff
@@ -1,20 +1,27 @@
 import { filterOnProps } from '../../utils/filterUtils';
 
-const object1 = {
+interface Props {
```

Comment:
I think creating types for mock data gives just more noise to the tests, and it's not that helpful (knowing that we need to pass prop1, prop2 etc. ).

I would remove this one.

Reply:
This interface is for fixing the error you see here: https://github.com/equinor/EchoUtils/runs/4896000365?check_suite_focus=true

Is there any better way to fix it?

Reply:
Ah, I see!
Well, I would go with `any` here, and that's it. I think Typescript should help us writing better code, especially when it comes to more complex structures.
In this actual case I think it's just hindering us. (Do we really care if this mock data has prop1, prop2 etc.) So, I would ignore typing here.

Reply:
Ok :) I changed it to any, and disabled the any check for this file.